### PR TITLE
Use price instead of price_tag

### PR DESCRIPTION
**Pull request by x4d3**
I think price should be send as a hash instead of a formatted string (because some information is lost, for example $12 can be 12  USD or 12 AUD)
See PR: https://github.com/maestrano/Maestrano/pull/1594

### CODE REVIEW

**Comment by x4d3** [comment]
@ouranos Could you please code review ?

File: core/app/pdf/mno_enterprise/invoice_pdf.rb [hunk 1]
```diff
@@ -116,6 +127,12 @@ def image_path(name)
       File.exists?(app_path) ? app_path : engine_path
     end
 
+    # Format a money object
+    def money(m)
+      "#{m.format(symbol: false)} #{m.currency_as_string}"
+    end
```
Comment (ouranos) [inline core/app/pdf/mno_enterprise/invoice_pdf.rb]: Why not just `m.format`? Is it to have the currency after the amount?

Comment (x4d3) [inline core/app/pdf/mno_enterprise/invoice_pdf.rb]: m.format is giving something like $ 8, that is ambiguous.

**Comment by ouranos** [comment]
Happy with this as long as @alachaum is happy with maestrano/Maestrano#1594 :)

File: core/app/pdf/mno_enterprise/invoice_pdf.rb (hunk 1 above)
Comment (ouranos) [inline core/app/pdf/mno_enterprise/invoice_pdf.rb]: Ah got it, `currency_as_string` will give USD / AUD

**Pull request merged**